viite-UI/src/utils/LinkValues.js

```diff
@@ -124,6 +124,19 @@
         Unknown:                        {value:99, description:"Ei määritelty"}
 
     };
+    root.RoadTypeDisplayValues = {
+        Empty:                          {value:0, description:"--"},
+        PublicRoad:                     {value:1, description:"1 Maantie"},
+        FerryRoad:                      {value:2, description:"2 Lauttaväylä maantiellä"},
+        MunicipalityStreetRoad:         {value:3, description:"3 Kunnan katuosuus"},
+        PublicUnderConstructionRoad:    {value:4, description:"4 Maantien työmaa"},
+        PrivateRoadType:                {value:5, description:"5 Yksityistie"},
+        UnknownOwnerRoad:               {value:9, description:"9 Omistaja selvittämättä"},
+
+
```

Comment:
we can also reuse existing RoadType and add some display value for dropdown for e.g. 

root.RoadType = {
        Undefined:                      {value:0, description:"Määrittelemätön", displayValue:"--"},
        PublicRoad:                     {value:1, description:"Yleinen tie", displayValue:"1 Maantie"},
        FerryRoad:                      {value:2, description:"Lauttaväylä yleisellä tiellä", displayValue:"2 Lauttaväylä maantiellä"},
        MunicipalityStreetRoad:         {value:3, description:"Kunnan katuosuus", displayValue:"3 Kunnan katuosuus"},
        PublicUnderConstructionRoad:    {value:4, description:"Yleisen tien työmaa", displayValue:"4 Maantien työmaa"},
        PrivateRoadType:                {value:5, description:"Yksityistie", displayValue:"5 Yksityistie"},
        UnknownOwnerRoad:               {value:9, description:"Omistaja selvittämättä", displayValue:"9 Omistaja selvittämättä"},
        Unknown:                        {value:99, description:"Ei määritelty", displayValue:"--"}
    };

Reply:
or a more sugestive name like displayText